src/Components/NavBar/menuData.ts

```diff
@@ -439,6 +439,10 @@ export const menuData: MenuData = {
       text: "Museums",
       href: "/institutions",
     },
+    {
+      text: "Consign",
+      href: "/consign",
+    },
```

Comment:
@xtina-starr So, I actually don't understand this at all: I'm only adding it because there's a spec for it — Why does this data exist if it's not actually used? And what's with the type casting in the component spec (LinkData => SimpleLinkData)?

Reply:
Just rebased and will merge on green. But if you know what's up with this — I'm interested. The duplication/spec doesn't make sense to me.

Reply:
I think the `menuData` file was a necessary part of an earlier iteration, but if it's not used then we should remove it + update the test!